src/js/popup.js

```diff
@@ -610,7 +610,15 @@ Logic.registerPanel(P_CONTAINERS_LIST, {
       const siteSettings = await Logic.getAssignment(currentTab);
       this.setupAssignmentCheckbox(siteSettings, currentTabUserContextId);
       const currentPage = document.getElementById("current-page");
-      currentPage.innerHTML = escaped`<span class="page-title truncate-text">${currentTab.title}</span>`;
+      //using DOMParser to modify innerHTML 
```

Comment:
@jonathanKingston - is `DOMParser` the best replacement for `innerHTML` assignment?

Reply:
Should I create a separate PR for the issues, I thought I could submit them both but since you are still assessing whether DOMParser is the best replacement what would you advise that I do? 
I also think  that it would make them count as 2 separate contributions on Outreachy. 
Thank you

Reply:
This shouldn't ever be used, it's not any safer than `innerHTML` in fact it breaks the linting which captures when we use `escaped`. I don't think this is a good idea at all.